g3/generate/pr/prompts/creator.py

```diff
@@ -1,8 +1,59 @@
 from typing import Optional
 
 from g3.domain.message_tone import MessageTone
+from g3.generate.pr.prompts.template import pr_template
+from g3.git.gitinfo import GitInfo
+from g3.main import config
 
 
 class Creator:
-    def create(self, tone: MessageTone, jira: Optional[str] = None, include: Optional[str] = None) -> list:
-        raise NotImplementedError
+    def __init__(self):
+        self.git_info = GitInfo()
+
+    def create(self, tone: MessageTone, include: Optional[str] = None) -> list:
```

Comment:
You missed the `jira` argument

Reply:
Jira should be included in the commit message so that's why I didn't add it here. Do you think we should also include it here?

Reply:
Yeap, `g pr` [supports](https://docs.google.com/document/d/1b3SkLVnrB_P-lYsD2Kv-fO-oUHciovQbTQbKsNswHtA/edit#heading=h.g330q4muzbfz) `--jira` option too